src/components/ResourceForm/index.jsx

```diff
@@ -0,0 +1,141 @@
+import { useState } from "react";
+import { useDropzone } from "react-dropzone";
+import { toast } from "react-hot-toast";
+import FileInput from "./FileInput";
+
+function ResourceForm() {
+  const [requiredLogoDetails, setRequiredLogoDetails] = useState({
+    name: "",
+    description: "",
+    default: null,
+    categoryId: null,
+  });
+
+  const [logoImagesByMode, setLogoImagesByMode] = useState({
+    darkmode: null,
+    "1.5x": null,
+    "2x": null,
+    "3x": null,
+    "4x": null,
+  });
+
+  const [previewSource, setPreviewSource] = useState(null);
```

Comment:
한 맥락의 useState 설정이기에 같이 붙여서 써줘도 되지 않을까 생각합니다 😺 
```suggestion
  const [requiredLogoDetails, setRequiredLogoDetails] = useState({
    name: "",
    description: "",
    default: null,
    categoryId: null,
  });
  const [logoImagesByMode, setLogoImagesByMode] = useState({
    darkmode: null,
    "1.5x": null,
    "2x": null,
    "3x": null,
    "4x": null,
  });
  const [previewSource, setPreviewSource] = useState(null);
```